CONTRIBUTING.md

```diff
@@ -138,7 +138,7 @@ An optional `--service` parameter can also be set to minimize the number of load
         "server",
         "start",
         "--service",
-        "<service-name>"
+        "<service-name-1>,<service-name-2>"
```

Comment:
Can you do a grep for --service and update the other references in README, TROUBLESHOOTING, docs/* as appropriate?

Reply:
I updated one more place in troubleshooting.md to give a hint on the possibility of starting the server with multiple services. Other than that, the existing single service examples are fine.